src/skai/beam_utils.py

```diff
@@ -84,17 +82,20 @@ def _get_setup_file_path():
   return str(pathlib.Path(__file__).parent.parent / 'setup.py')
 
 
+
```

Comment:
nit: Can you remove these empty lines at 85 and 87? Our internal linter will complain :P